javacutil/src/main/java/org/checkerframework/javacutil/ToStringComparator.java

```diff
@@ -0,0 +1,37 @@
+// TODO: Put this in plume-util and use it from there.
+package org.checkerframework.javacutil;
+
+import java.util.ArrayList;
+import java.util.Comparator;
+import java.util.List;
+import java.util.Objects;
+
+/** A comparator that orders values based on the lexicographic ordering of their toString(). */
+public class ToStringComparator implements Comparator<Object> {
```

Comment:
Should this be `Comparator<@Nullable Object>`?

Reply:
It implements both `Comparator<Object>` and `Comparator<@Nullable Object>`.  I think I can only name one of those, and the two are incompatible.  Should I create a parameterized `ToStringComparator` instead?

Reply:
The type argument of `Comparator` should be contravariant. That way we could make this `Comparator<@Nullable Object>` which would be a subtype of `Comparator<@NonNull Object>` (We don't have the functionality; #1970 is a request to implement it.)

So for now, I think it's enough to mention that ToStringComparator, handles `null` elements in that class Javadoc.

As an aside, the Nullness Checker line 34 should have issued a warning on line 34, I'll look into why it didn't.

Reply:
@smillst Let me know if the new comment looks good.

Reply:
Looks good thanks!